kube/src/api/metadata.rs

```diff
@@ -2,25 +2,26 @@ pub use k8s_openapi::apimachinery::pkg::apis::meta::v1::{ListMeta, ObjectMeta};
 use k8s_openapi::Metadata;
 use serde::{Deserialize, Serialize};
 
-/// An accessor trait for Metadata
+/// An accessor trait for Metadata.
+///
+/// This is for a subset of Kubernetes type that do not end in `List`.
+/// These types, using [`ObjectMeta`], SHOULD all have required properties:
+/// - `.metadata`
+/// - `.metadata.name`
 ///
-/// This for a subset of Kubernetes type that do not end in List
-/// These types, using ObjectMeta, SHOULD all have required properties:
-/// - .metadata
-/// - .metadata.name
 /// And these optional properties:
-/// - .metadata.namespace
-/// - .metadata.resource_version
+/// - `.metadata.namespace`
```

Comment:
The lack of period made the sentence confusing. Also, the list wasn't rendered correctly:
![image](https://user-images.githubusercontent.com/639336/103120475-c3a7ee00-462c-11eb-944c-578b69d7f089.png)

Fixed version:
![image](https://user-images.githubusercontent.com/639336/103120484-ce628300-462c-11eb-8146-f806f27ea72a.png)